pyvista/core/filters.py

```diff
@@ -4326,6 +4326,42 @@ def strip(poly_data, join=False, max_length=1000, pass_cell_data=False,
         alg.Update()
         return _get_output(alg)
 
+    def shrink(poly_data, shrink_factor=1.0, inplace=False, progress_bar=False):
+        """Shrink a mesh.
+
+        This helps shrink a mesh.
+
```

Comment:
Docstring could be improved:

```suggestion
        """Shrink the individual faces of a mesh.

        This filter shrinks the individual faces of a mesh rather than scaling the entire mesh.

```